.gitignore

```diff
@@ -157,4 +157,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+.idea/
```

Comment:
Doesn't this belong to your [global .gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)?

Reply:
I actually didn't realize this was possible, so I reverted that commit.🫠